### PR TITLE
wire up migrate V1 to V2 REP

### PR DESCRIPTION
**Pull request by andrewdamelio**

### CODE REVIEW

**Review by ghost — COMMENTED** [review]
This pull request has been automatically deployed to [FeaturePeek](https://dashboard.featurepeek.com). 🍻
Your deployment will be kept up-to-date with this pull request's latest changes.

**https://peek.run/jvgho9z**

-------

*Please read [our docs](https://docs.featurepeek.com/configuring-peek-yml/) for more configuration details.*

File: packages/augur-ui/src/modules/app/components/side-nav/side-nav.styles.less [hunk 1]
```diff
@@ -109,6 +109,10 @@
         }
       }
     }
+
+    > div {
+      margin-left: 32px;
```
Comment (bthaile) [inline packages/augur-ui/src/modules/app/components/side-nav/side-nav.styles.less]: we have a constant for 32px

Comment (andrewdamelio) [reply packages/augur-ui/src/modules/app/components/side-nav/side-nav.styles.less]: whoops

**Review by ghost — COMMENTED** [review]
This pull request has been automatically deployed to [FeaturePeek](https://dashboard.featurepeek.com). ✌️
Your deployment will be kept up-to-date with this pull request's latest changes.

**https://peek.run/l74hpx2**

-------

*Please read [our docs](https://docs.featurepeek.com/configuring-peek-yml/) for more configuration details.*

**Review by bthaile — APPROVED** [review]
works, lgtm

**Pull request merged**